lnwallet/test_utils.go

```diff
@@ -7,6 +7,7 @@ import (
 	"encoding/hex"
 	"io"
 	"io/ioutil"
+	rrand "math/rand"
```

Comment:
Nit: this is usually `prand` within the codebase.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/3016/commits/d390428983891065fd8c6cbb2bd3af625b548e08